pkg/importer/dataStream.go

```diff
@@ -37,59 +38,59 @@ type DataStreamInterface interface {
 	Close() error
 }
 
-var _ DataStreamInterface = &dataStream{}
+var _ DataStreamInterface = &DataStream{}
 
-// implements the ReadCloser interface
-type dataStream struct {
-	Url         *url.URL
-	Readers     []Reader
+// DataStream implements the ReadCloser interface
+type DataStream struct {
+	url         *url.URL
+	Readers     []reader
 	buf         []byte // holds file headers
 	qemu        bool
 	Size        int64
-	accessKeyId string
+	accessKeyID string
 	secretKey   string
 }
 
-type Reader struct {
-	RdrType int
-	Rdr     io.ReadCloser
+type reader struct {
+	rdrType int
+	rdr     io.ReadCloser
 }
 
 const (
-	RdrHttp = iota
-	RdrS3
-	RdrFile
-	RdrGz
-	RdrMulti
-	RdrTar
-	RdrXz
+	rdrHTTP = iota
+	rdrS3
+	rdrFile
+	rdrGz
+	rdrMulti
+	rdrTar
+	rdrXz
 )
 
-// map scheme and format to RdrType
+// map scheme and format to rdrType
 var rdrTypM = map[string]int{
-	"gz":    RdrGz,
-	"http":  RdrHttp,
-	"https": RdrHttp,
-	"local": RdrFile,
-	"s3":    RdrS3,
-	"tar":   RdrTar,
-	"xz":    RdrXz,
+	"gz":    rdrGz,
+	"http":  rdrHTTP,
+	"https": rdrHTTP,
+	"local": rdrFile,
+	"s3":    rdrS3,
+	"tar":   rdrTar,
+	"xz":    rdrXz,
 }
 
-// Return a dataStream object after validating the endpoint and constructing the reader/closer chain.
+// NewDataStream returns a DataStream object after validating the endpoint and constructing the reader/closer chain.
 // Note: the caller must close the `Readers` in reverse order. See Close().
-func NewDataStream(endpt, accKey, secKey string) (*dataStream, error) {
+func NewDataStream(endpt, accKey, secKey string) (*DataStream, error) {
 	if len(accKey) == 0 || len(secKey) == 0 {
-		glog.V(Vadmin).Infof("%s and/or %s are empty\n", IMPORTER_ACCESS_KEY_ID, IMPORTER_SECRET_KEY)
+		glog.V(2).Infof("%s and/or %s are empty\n", cdi.IMPORTER_ACCESS_KEY_ID, cdi.IMPORTER_SECRET_KEY)
```

Comment:
yeah I know I know :)  I've been trying to do these initial changes package per package, that's caused some confusion with glog so I've since submitted a patch that just updated all of the glog calls in one patch. PR #408